tests/robot-tests/tests/general_public/table_tool_absence_in_prus.robot

```diff
@@ -20,36 +20,37 @@ Select "Pupil absence" publication
     user opens details dropdown    Pupil absence
     user selects radio      Pupil absence in schools in England
     user clicks element    css:#publicationForm-submit
-    user waits until page contains    Choose a subject
+    user waits until element is visible  xpath://h2[text()="Choose a subject"]
```

Comment:
Would change this to `user waits until page contains heading 2`, then change the keyword (and the other heading keywords) to check on visibility.

Do the same for the rest of the xpath selectors.